src/components/Icon/ProfileAvatar/index.tsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import type { CSSProperties } from 'react';
 
 import { gray150 } from '../../../core/GlobalStyle';
 
 import { Container, Svg } from './ProfileAvatar.styles';
 
-export const ProfileAvatar = ({ className, color = gray150, width = 68 }) => (
+interface Props {
+  className?: string;
+  color?: CSSProperties['color'];
+  width?: number;
+}
+
+export const ProfileAvatar = ({
+  className,
+  color = gray150,
```

Comment:
```suggestion
  color = contents200,
```
gray150 이제 안쓰기로 했던것같아서 마이그레이션 가이드 따라 요렇게 바꿈 어떨까요

Reply:
@yongsk0066 슬랙에 gray150 검색해봤을 때 언급해주신 마이그레이션 가이드를 찾을 수 없어서 혹시 관련 링크가 있을까요?

Reply:
https://github.com/dramancompany/remember-ui/releases/tag/v2.0.0

Reply:
@qkraudghgh @yongsk0066 살펴보니 아직 남아있는 old color가 많은거 같은데 일괄 수정 하는게 좋을거 같아서 별도 PR로 분리하여 수정하도록 하겠습니다

Reply:
https://github.com/dramancompany/remember-ui/wiki/v2.0.0-%EB%A7%88%EC%9D%B4%EA%B7%B8%EB%A0%88%EC%9D%B4%EC%85%98-%EA%B0%80%EC%9D%B4%EB%93%9C
요깃습니다!

Reply:
아 늦었네